src/templates/main.tsx

```diff
@@ -41,11 +41,11 @@ export const getPath: GetPath<TemplateProps> = ({ document }) => {
     // temporary: guard for generated repo-based static page
     return `static-${Math.floor(Math.random() * (10000 - 1))}`;
   }
-  return document.slug
-    ? document.slug
-    : `${document.locale}/${document.address.region}/${document.address.city}/${
+  return document.address
```

Comment:
You need to update the template-manifest.json file as well to reflect the template name change so that the code template can be found for the pageset during build. If not this is breaking.